kv/local_sender.go

```diff
@@ -157,7 +159,13 @@ func (ls *LocalSender) Send(ctx context.Context, call proto.Call) {
 			trace.Event("read has no clock uncertainty")
 			header.Txn.MaxTimestamp = header.Txn.Timestamp
 		}
-		err = store.ExecuteCmd(ctx, call)
+		reply, err = store.ExecuteCmd(ctx, call.Args)
```

Comment:
Ah, I pointed to the wrong place. It was actually here https://github.com/cockroachdb/cockroach/pull/1729/files#diff-dae9f8a58ca2ec436440b12989c1982dR1253